src/transformers/models/marian/convert_marian_to_pytorch.py

```diff
@@ -532,7 +536,7 @@ def __init__(self, source_dir, eos_token_id=0):
         check_marian_cfg_assumptions(cfg)
         self.hf_config = MarianConfig(
             vocab_size=cfg["vocab_size"],
-            decoder_vocab_size=cfg.get("decoder_vocab_size", cfg["vocab_size"]),
+            decoder_vocab_size=cfg.get("decoder_vocab_size", cfg["decoder_vocab_size"]),
```

Comment:
Not sure this will work, `cfg["decoder_vocab_size"])` is always gonna be evaluated